File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/SensoryPerceptionAssessmentFormViewModel.cs
//$B1F84967
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class SensoryPerceptionAssessmentFormServiceController
    {
    }
}

namespace Core.Models
{
    public partial class SensoryPerceptionAssessmentFormViewModel : BaseAdditionalInfoFormViewModel
    {
        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
            var sensoryPerceptionAssessmentFormService = new SensoryPerceptionAssessmentFormServiceController();
            sensoryPerceptionAssessmentFormService.SensoryPerceptionAssessmentFormInternal(this, objectContext);
            return (SensoryPerceptionAssessmentForm)objectContext.GetObject(this._SensoryPerceptionAssessmentForm.ObjectID, "SensoryPerceptionAssessmentForm");
        }
    }
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/SensoryPerceptionAssessmentFormViewModel.ts
//$1A41A56B
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { SensoryPerceptionAssessmentForm } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class SensoryPerceptionAssessmentFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.SensoryPerceptionAssessmentFormViewModel, Core';
    @Type(() => SensoryPerceptionAssessmentForm)
    public _SensoryPerceptionAssessmentForm: SensoryPerceptionAssessmentForm = new SensoryPerceptionAssessmentForm();
}
